Program.cs

```diff
@@ -240,6 +239,8 @@ public static object DoStuff(LMRItemTracker.LaMulanaItemTrackerForm laMulanaItem
                 {
                     System.Console.WriteLine(ex.StackTrace);
                 }
+
+                Thread.Sleep(sleeptarget - DateTime.UtcNow);
```

Comment:
What if things get delayed somehow and we wind up with a negative number here? That's cause an error (System.ArgumentOutOfRangeException, apparently)